lib/index.js

```diff
@@ -80,9 +76,12 @@ module.exports = function (options) {
     tu.make_client(make_send, client_options, clientdone)
 
     function make_send (spec, topic, send_done) {
+      // Topic can be something like "pin:something" to match listener pin
+      if (args.topic) topic = 'seneca_' + args.topic.replace(':', '_') + '_';
```

Comment:
^ This allows you to pass in a "topic" to the client.  The server chooses its topic based on what you pin but this doesn't work for the client... so have to override it like this in the transport implementation.